### DISCUSSION TITLE
Image resizing

### DISCUSSION THREAD

**Discussion by njugunamwangi** [awcodes/filament-tiptap-editor#338] [General]
I am using curator media action in tiptap markdown and I was wondering if it is possible to resize the image

**Comment by awcodes** [comment]
Resize the image in what way?

**Reply by njugunamwangi** [reply]
https://drive.google.com/file/d/1CVgRLqlRKRrChvpbs6IGmyHPupP-oYpk/view?usp=sharing

**Reply by awcodes** [reply]
Says I don't have access to view the file.

**Reply by njugunamwangi** [reply]
https://drive.google.com/file/d/1IhkTDFnxoKR5f8kLfqnaUxy9axXkJEHM/view?usp=sharing

Sorry for that

**Reply by awcodes** [reply]
You can set the width and height when uploading or editing the uploaded image. But the drag resizing isn't built in. Nor have I seen a Tiptap extension for it, but it might exist. Not something at the moment that I have time to explore for core though.

**Comment by gacbaluyot** [comment]
is there any update on this one?
